tests/web/test_main.py

```diff
@@ -438,30 +437,30 @@ def test_fetchdf(web_sushi_context: Context) -> None:
     assert not df.empty
 
 
-def test_get_models(web_sushi_context: Context) -> None:
-    # TODO: add better tests for this endpoint
-    response = client.get("/api/models")
-    json_models = [
-        models.Model(
-            name=model.name,
-            path=str(model._path.relative_to(web_sushi_context.path)),
-            description=model.description,
-            owner=model.owner,
-            dialect=model.dialect,
-            columns=[
-                models.Column(
-                    name=name,
-                    type=str(data_type),
-                    description=model.column_descriptions.get(name),
-                )
-                for name, data_type in model.columns_to_types.items()
-            ],
-        ).dict()
-        for model in web_sushi_context.models.values()
-    ]
-
-    assert response.status_code == 200
-    assert json_models == response.json()
+# TODO: add better tests for this endpoint
```

Comment:
I think we can simplify this test. There's not a lot of complex logic in this endpoint, mostly just serializing fields. Maybe update this test to check the length of the payload and a serialized model (could just be the first model in the array) contains important fields like details, name, columns, etc. and whatever else you think should be tested, like maybe model type is correct?